graylog2-server/src/main/java/org/graylog/plugins/views/search/jobs/SearchJobType.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.plugins.views.search.jobs;
+
+public enum SearchJobType {
+    DATA_LAKE,
+    //SEARCH_ENGINE - TBD in future
```

Comment:
we call the search engine "Indexer" in other places - or do you think that "Indexer" as a name/concept would also include the Data Lake?

Reply:
I will reflect this naming dilemma in the comment, we will get to refactoring names a little bit later, probably.